lms/djangoapps/instructor/views/api.py

```diff
@@ -3275,8 +3269,8 @@ def re_validate_certificate(request, course_key, generated_certificate):
         certificate_invalidation.deactivate()
 
     # We need to generate certificate only for a single student here
-    students = [certificate_invalidation.generated_certificate.user]
-    instructor_task.api.generate_certificates_for_students(request, course_key, students=students)
+    student = [certificate_invalidation.generated_certificate.user]
+    instructor_task.api.generate_certificates_for_students(request, course_key, specific_student=student)
```

Comment:
@adampalay we use ```id__in``` at django filter objects to find students no matter it only one or more than one so thats why i am still using this.